Content.Client/Chat/Managers/IChatManager.cs

```diff
@@ -1,9 +1,15 @@
 using Content.Shared.Chat;
+using Content.Shared.Chat.Prototypes;
+using Content.Shared.Radio;
 
 namespace Content.Client.Chat.Managers
 {
     public interface IChatManager : ISharedChatManager
     {
-        public void SendMessage(string text, ChatSelectChannel channel);
+        public void SendMessage(string text, ChatSelectChannel channel, RadioChannelPrototype? radioChannel);
```

Comment:
but this ine is cursed :pepeHands: why do we leave it be and not kill it with fre? it is called RadioChannel but is changeling hiveming radio channel? thats all so confusing so i'd be better off without it... but oh well...